prolif/plotting/network.py

```diff
@@ -910,15 +803,7 @@ def save_png(self) -> Any:
         .. versionadded:: 2.1.0
         """
         return display(
-            Javascript(f"""
-            var iframe = document.getElementById("{self.uuid}");
-            var iframe_doc = iframe.contentWindow.document;
-            var canvas = iframe_doc.getElementsByTagName("canvas")[0];
-            var link = document.createElement("a");
-            link.href = canvas.toDataURL();
-            link.download = "prolif-lignetwork.png"
-            link.click();
-            """),
+            Javascript(f"saveAsPNG('{self.uuid}')"),
```

Comment:
unfortunately this doesn't work as the `saveAsPNG` function is defined within an iframe which `Javascript` calls can't access. It's a small enough function that I'm ok with keeping it here as is instead of the network.js file.